src/response/cache/sorted_set_fetch.rs

```diff
@@ -0,0 +1,138 @@
+use std::convert::{TryFrom, TryInto};
+
+use momento_protos::cache_client::sorted_set_fetch_response::found::Elements;
+use momento_protos::cache_client::sorted_set_fetch_response::SortedSet;
+use momento_protos::cache_client::SortedSetFetchResponse;
+
+use crate::{
+    response::{MomentoErrorCode, SdkError},
+    MomentoError, MomentoResult,
+};
+
+// TODO this needs to be moved to the requests directory
```

Comment:
i think i moved this in my PR that got merged, and it looks like maybe during rebase you re-introduced it?

Reply:
oh whoops, will fix that